plugins/Contents/tests/Fixtures/TwoVisitsWithContents.php

```diff
@@ -47,7 +47,7 @@ public function trackVisits()
 
         $this->trackContentImpressionsAndInteractions($vis);
 
-        $this->dateTime = Date::factory($this->dateTime)->addHour(0.5);
+        $this->dateTime = Date::factory($this->dateTime)->addHour(0.5)->getDatetime();
```

Comment:
we used to pass a `Date` object here meaning the tracker would do `$date->toString()` which then would only result in `YYYY-MM-DD` and throw away the entire time. So this was before actually not working which is why some tests are failing.